packages/nextra-theme-docs/src/types.ts

```diff
@@ -94,7 +94,8 @@ export interface DocsThemeConfig {
     labels: string
   }
   sidebar: {
-    defaultMenuCollapsed: boolean
+    defaultMenuCollapseLevel: number,
+    defaultMenuCollapsed: boolean,
```

Comment:
it's a definitely nice feature, but let's stay with only one option to configure this behaviour - `defaultMenuCollapseLevel: number`, `defaultMenuCollapsed` should be removed